modules/PersonalBank.py

```diff
@@ -14,10 +14,12 @@ class PersonalBank:
     """
 
     
-    def __init__(self, balance, history_of_transactions):      
-    
-        self.balance = balance
-        self.history_of_transactions = history_of_transactions
+    def __init__(self, balance = None):      
```

Comment:
You could literally write just `balance=350` and not use the if statement in the body.

Reply:
Maybe in case of putting a different `balance` value. Seems like a better thing to me.